okhttp/src/main/java/okhttp3/internal/UtilKt.kt

```diff
@@ -57,13 +57,23 @@ inline fun ignoreIoExceptions(block: () -> Unit) {
   }
 }
 
+inline fun threadName(name: String, block: () -> Unit) {
+  val oldName = Thread.currentThread().name
```

Comment:
Is `Thread.currentThread()` expensive? Could make a local

Reply:
Fixed.